docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst

```diff
@@ -0,0 +1,124 @@
+Create a MQTT source connector for RabbitMQ®
```

Comment:
Can we call this `Create a MQTT source connector` and then just say in the text that we're taking the RabbitMQ eexample

Reply:
done